### PR TITLE
added admin decision and contact person functionality

### PR DESCRIPTION
**Pull request by 0xonaroll**
The initial commit was made before the merge of a different PR, so it was reverted then that PR was merged and then the reverted commit was reverted again and pushed to this new branch and is now in this new PR.

### CODE REVIEW

File: client/assets/js/admin-controller.js [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+        'status': 'status',
+        'contact': 'contact'
     };
 
     // sorting state
     $scope.sortKey = 'pointDisplay';
     $scope.sortReverse = true;
 
+
+    // store projex members
```
Comment (Anton-Cao) [inline client/assets/js/admin-controller.js]: projex -> projx

File: client/assets/js/admin-controller.js [hunk 2]
```diff
@@ -64,4 +73,72 @@ angular.module('portal').controller('adminController', function ($scope, $http,
...
+
+        $scope.projects.forEach(function(project) {
+
+            $scope.projectMatches = 0;
+            $scope.projectsOldState.forEach(function(projectOld) {
+                if (project._id === projectOld._id) {
+                    $scope.projectMatches += 1;
+                    if (project.private.status !== projectOld.private.status) {
+                        //$scope.statusChangeUpdater(project);
+                    };
```
Comment (Anton-Cao) [inline client/assets/js/admin-controller.js]: remove empty if block

File: client/assets/js/admin-controller.js [hunk 3]
```diff
@@ -64,4 +73,72 @@ angular.module('portal').controller('adminController', function ($scope, $http,
...
+                }
+            });
+            
+            if ($scope.projectMatches !== 1) {
+                $internalError = true;
+                console.log('Project states are not alligned correctly');
+            }
+
+            // project.private.status = project.private.status.toLowerCase();
+            // project.private.contact = project.private.contact.toLowerCase();
```
Comment (Anton-Cao) [inline client/assets/js/admin-controller.js]: remove comments

File: client/views/home.html [hunk 4]
```diff
@@ -42,7 +42,9 @@ <h2>your projects</h2>
                 <div class="ui right floated">
                   <div class="ui tag projx label" ng-show="project.private.batch==='ProjX {{ cycle }}'">ProjX {{ cycle }}</div>
                   <div class="ui tag yellow label" ng-show="project.display.complete==='yes'">Application complete</div>
```
Comment (Anton-Cao) [inline client/views/home.html]: could consider changing "Application complete" to "Application awaiting review"

File: client/views/project.html [hunk 5]
```diff
@@ -10,7 +10,9 @@
 
         <div class="ui tag projx label" ng-show="projectDisplay.batch==='ProjX {{ cycle }}'">ProjX {{ cycle }}</div>
         <div class="ui tag yellow label" ng-show="projectDisplay.complete==='yes'">Application complete</div>
```
Comment (Anton-Cao) [inline client/views/project.html]: same as above

File: client/assets/js/admin-controller.js [hunk 6]
```diff
@@ -1,21 +1,30 @@
-angular.module('portal').controller('adminController', function ($scope, $http, $location) {
+angular.module('portal').controller('adminController', function ($scope, $http, $location, $routeParams) {
```
Comment (Anton-Cao) [inline client/assets/js/admin-controller.js]: why do we need `routeParams` here?

File: server/routes/user.js [hunk 7]
```diff
@@ -166,5 +166,15 @@ router.get('/countdown', function(req,res,next) {
 });
 
 
+/**
+ * GET /projxTeam - gets projx team members from config
+ */
+router.get('/projxTeam', function(req,res,next) {
+    res.status(200).send({
+        "team": config.projxTeam
```
Comment (Anton-Cao) [inline server/routes/user.js]: add `projxTeam` to example `config.json` in readme

File: server/routes/user.js [hunk 8]
```diff
@@ -156,7 +156,7 @@ router.post('/update', perm.user, function(req, res) {
 });
 
 /**
- * POST /countdown - gets event name and time from config
+ * GET /countdown - gets event name and time from config
  */
 router.get('/countdown', function(req,res,next) {
```
Comment (Anton-Cao) [inline server/routes/user.js]: spaces between the method parameters

File: server/routes/user.js [hunk 9]
```diff
@@ -166,5 +166,15 @@ router.get('/countdown', function(req,res,next) {
 });
 
 
+/**
+ * GET /projxTeam - gets projx team members from config
+ */
+router.get('/projxTeam', function(req,res,next) {
```
Comment (Anton-Cao) [inline server/routes/user.js]: spaces between the method parameters

**Review by Anton-Cao — CHANGES_REQUESTED** [review]
Good work! Left some minor comments.

File: client/assets/js/admin-controller.js [hunk 10]
```diff
@@ -83,9 +83,9 @@ angular.module('portal').controller('adminController', function ($scope, $http,
             $scope.projectsOldState.forEach(function(projectOld) {
                 if (project._id === projectOld._id) {
                     $scope.projectMatches += 1;
-                    if (project.private.status !== projectOld.private.status) {
-                        //$scope.statusChangeUpdater(project);
-                    };
+                    // if (project.private.status !== projectOld.private.status) {
+                    //     $scope.statusChangeUpdater(project);
+                    // };
```
Comment (Anton-Cao) [inline client/assets/js/admin-controller.js]: will this be used in the future? if not, the comments can be removed

**Review by Anton-Cao — CHANGES_REQUESTED** [review]
Just left one small comment.

**Comment by 0xonaroll** [comment]
admin decision privileges and assignment of contact person

**Pull request merged**